package.json

```diff
@@ -51,6 +53,7 @@
     "html-react-parser": "^1.2.4",
     "json-loader": "^0.5.1",
     "lodash": "^4.17.15",
+    "mocha": "^8.4.0",
```

Comment:
❓ I'm not very familiar with how dependencies work, so I could be way off the mark with this, but since mocha is a test framework, should it be in the `devDependencies`?

Reply:
ya  I wasn't sure if the github action environment would install devDependencies, but looks like it does 👍  moved
https://github.com/stellar/dashboard/pull/188/commits/cc7de972a00f5134ffdd4bc92cca6ceffcfb27c4